infra/modules/secret/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "manage_method" {
+  type        = string
+  description = <<EOF
+    Method to manage the secret. Options are 'manual' or 'code'.
+    Set to 'code' to generate a random secret.
+    Set to 'manual' to reference a secret that was manually created and stored in AWS parameter store.
+    Defaults to 'code'."
+    EOF
```

Comment:
Oo. I like that you're using a multi-line description because it allows us to better describe what the variables. 

Nit: I would suggest not using `EOF` because that implies "end of file" but something like `EOT` (which is used in the [terraform docs](https://developer.hashicorp.com/terraform/language/expressions/strings)) instead.